{"name": "test_login_0_wrongPwd_noReg[phone_pwd_data3]", "status": "broken", "statusDetails": {"message": "TypeError: 'function' object is not subscriptable", "trace": "@pytest.fixture(scope='class')\n    def access_web():\n        '''The scope for which this fixture is shared; one of ``\"function(setUp/teardown)\"``\n                (default), ``\"class(setUpClass/teardownClass)\"``, ``\"module\"``, ``\"package\"`` or ``\"session\"``.'''\n        global driver\n        # 前置操作\n        my_log.info(\"====所有用例的前置操作：初始化浏览器对话，登录前程贷系统(setUp)====\")\n        driver = webdriver.Chrome()\n        driver.maximize_window()\n        driver.get(CD.web_login_url)\n        lg = LoginPage(driver)\n>       yield (driver, lg)  # 分割线，返回值\nE       TypeError: 'function' object is not subscriptable\n\nTestCases\\conftest.py:33: TypeError"}, "parameters": [{"name": "phone_pwd_data", "value": "{'user': '17307398595', 'password': 'test123456', 'check': '此账号没有经过授权，请联系管理员!', 'title': '异常用例-账号未授权'}"}], "start": 1646142704060, "stop": 1646142704060, "uuid": "a460a672-6a22-45d5-8467-eccc04727e68", "historyId": "2113d70c41b9d747ef747148c3cb4c1f", "testCaseId": "5cd5d761a01b5217fcaf5e12ea3f20b7", "fullName": "TestCases.test_00_login.TestLogin#test_login_0_wrongPwd_noReg", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('refresh_page')"}, {"name": "parentSuite", "value": "TestCases"}, {"name": "suite", "value": "test_00_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "DESKTOP-KL6VQQI"}, {"name": "thread", "value": "3856-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "TestCases.test_00_login"}]}